trivy-secret.yaml

```diff
@@ -83,6 +83,34 @@ rules:
     regex: (?P<secret>UID2-O-P-.{6}\..{38})
     secret-group-name: secret
 
+  ########################################
+  # UID2 Client Side Keypair Private Key #
+  ########################################
+  - id: uid2-client-side-keypair-private-key-test
+    category: uid2
+    title: UID2 - Client Side Keypair Private Key - Test
+    severity: CRITICAL
+    keywords:
+      - UID2-Y-T
+    regex: (?P<secret>UID2-Y-T-.{92})
```

Comment:
It's possible that we will have keys of different lengths, e.g. if we use a different curve. Can we keep the prefix and leave the length unspecified?

Reply:
hmm I'm not sure how to do an unspecified length if the keys don't have a known terminating character/sequence.